src/style.css

```diff
@@ -0,0 +1,56 @@
+.search__bar {
+  position: relative;
+  display: flex;
+  align-items: center;
+}
+
+.search__bar--img {
+  width: 19px;
+  height: 19px;
+  position: absolute;
+  left: 26px;
+}
+
+.search__bar--field {
+  line-height: 70px;
+  color: #3d4e61;
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 350;
+  width: 900px;
+  height: 70px;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background: #fff;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  display: flex;
+  align-items: center;
+  padding-left: 62px;
+}
+
+.search__bar--field:hover {
+  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.2);
+}
+
+.search__bar--field:focus {
+  border-radius: 3px;
+  border: 1px solid #e1e7ed;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  font-weight: 800;
+}
+
+.search__barSmall--img {
+  width: 11px;
+  height: 11px;
+  position: absolute;
+  left: 13px;
+}
+
+.search__barSmall--field {
+  width: 427px;
+  height: 42px;
+  line-height: 42;
+  font-size: 14px;
+  padding-left: 33px;
+}
```

Comment:
You already have `search__bar--small` style on the wrapper div, so you can use it to alter the styles of `search__bar--img` and `search__bar--input`. In fact, it will remove the need for additional classes (`search__barSmall--img` and  `search__barSmall--field`)